features/support/branch_helpers.rb

```diff
@@ -57,6 +71,16 @@ def existing_remote_branches
 end
 
 
+# Returns the name of the given branch if it was local
+#
+# 'foo'          --> 'foo'
+# 'origin/foo'   --> 'foo'
+# 'upstream/foo' --> 'foo'
+def local_branch_name branch_name
+  branch_name.sub(/.+\//, '')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.